_extras/customization.md

```diff
@@ -163,17 +224,25 @@ please follow the steps bellow:
 
 1.  Add the workshop-template repository as upstream:
 
-        $ git remote add upstream https://github.com/carpentries/workshop-template.git
+```bash
+$ git remote add upstream https://github.com/carpentries/workshop-template.git
+```
 
 2.  Fetch the data from upstream repository (also know as the workshop-template
     repository):
 
-        $ git pull upstream
+```bash
+$ git pull upstream
+```
 
 4.  Address possible merge conflicts, and
 
-        $ git commit -a
+```bash
+$ git commit -a
+```
 
 5.  Push the changes to your repository on GitHub:
 
-        $ git push origin gh-pages
+```bash
+$ git push origin gh-pages
+```
```

Comment:
Shouldn't these be standard Carpentries code blocks?
```
~~~
...
~~~
{: .language-bash}
```

I also suggest adding `{% include links.md %}` to the end of this document.

Reply:
:+1: